tests/bluetooth/audio/ascs/mocks/ascs_help_utils.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
Is the header guard left out intentionally as this is sort of an internal file? I can see the value in getting the compilation errors if the wrong file includes this, but it also weighs to not see the guard :)

Reply:
@fredrikdanebjer You know, I end up with guards. It's because I had to include one of the headers in other header file, which caused compilation error for me ;)